test/build.gradle.kts

```diff
@@ -39,17 +46,17 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
     }
 
     kotlinOptions {
-        jvmTarget = "1.8"
+        jvmTarget = JavaVersion.VERSION_11.toString()
```

Comment:
I like the following syntax better:
jvmTarget = "${JavaVersion.VERSION_11}"
What do you think?

Reply:
I've asked experienced developer about these 3 different approaches:

1. `        jvmTarget = JavaVersion.VERSION_11.toString()`
2. `        jvmTarget = "$JavaVersion.VERSION_11"
3.         jvmTarget = "11"

This is what he said:

>They're all functionally equivalent. The first is just getting the name from a concrete type as opposed to a "magic string" like the third, and the second technically may be less efficient because it involves string interpolation. None of it matters in practice for a build script though

So, should I change it?

Reply:
It is not necessary